irc/io_modules/alert/irc.lua

```diff
@@ -0,0 +1,65 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+--[[
+# Heka IRC Alert Output Module
+
+## Sample Configuration
+
+```lua
+alert = {
+    modules = {
```

Comment:
This will only support a single channel I suspect we will want different message to go to different channels (possibly one message to multiple channels).  The channel selection should be part of the alert cfg in irc/modules/heka/alert/irc.lua and this cfg will define all of the acceptable endpoints

Reply:
The underlying c code only supports a single channel. This was done intentionally to keep the state machine in that code simpler. My thought was that if different output was required, it would just be another instance of the alert module with a different configuration (and a separate connection).

If we want to support channel configuration here, I think we should also look at modifying `irc.c` to support managing multiple channels if you prefer this.